### ISSUE TITLE
qwen3vl-8b instruct sft video performance drop

### ISSUE THREAD

**Issue by xiaoqqqqqqq4999** [modelscope/ms-swift#8409]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new question or discussion topic. / 我已经搜索过现有的 issues，确认这是一个新的问题与讨论。

### Question Description / 问题描述

I noticed a significant performance drop when using SWIFT for video SFT. I tried adjusting the learning rate, freezing the ViT, and modifying the data, but nothing worked. Finally, I used 100k samples from the llava-video-178k dataset; after training for only 10 steps, the performance on VideoMME dropped by about 15 points. Is this normal, or is there a bug in the video training process? Note that these QA pairs only contain the answer options.

**Comment by Jintao-Huang** [comment]
Check if the issue is caused by inconsistent preprocessing between training and inference.

**Comment by Jintao-Huang** [comment]
plesase share the shell script

**Comment by xiaoqqqqqqq4999** [comment]
Thank you very much for your reply. This is my script: 
### **Environment & Setup**

| Component | Version/Config |
|-----------|--------------|
| Model | `Qwen3VL-8B-Instruct` |
| GPUs | 128 × NVIDIA GPUs (16 nodes × 8 GPUs) |
| Training Type | Full Fine-tuning (`--train_type full`) |
| Precision | `bfloat16` |
| DeepSpeed | ZeRO-2 |
| Attention | Flash Attention 2 |
| Batch Size | 1 per device × 2 gradient accumulation = effective batch size 256 |
| MAX_MODEL_SEQ | 16384 (in qwen_vl_utils to set total video pixels) | 
---

### **Training Script**

```bash
export NPROC_PER_NODE=8
export MASTER_PORT=$MASTER_PORT
export NNODES=$NNODES
export NODE_RANK=$RANK
export MASTER_ADDR=$MASTER_ADDR
export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7

# Video/Token Config
export IMAGE_MAX_TOKEN_NUM=1024
export VIDEO_MAX_TOKEN_NUM=512
export VIDEO_MIN_TOKEN_NUM=64
export FPS_MIN_FRAMES=4
export FPS_MAX_FRAMES=128

# Dependencies
pip uninstall decord -y
pip install torchcodec==0.6.0
pip install "numpy<2.0"

# Dataset
dataset="llava_video_165k.jsonl"

swift sft \
    --model Qwen3VL-8B-Instruct \
    --check_model False \
    --train_type full \
    --freeze_llm false \
    --freeze_aligner true \
    --freeze_vit true \
    --dataset $dataset \
    --torch_dtype bfloat16 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 1 \
    --learning_rate 5e-6 \
    --gradient_accumulation_steps 2 \
    --output_dir swift_sft \
    --save_steps 600 \
    --save_total_limit 5 \
    --logging_steps 1 \
    --max_length 36000 \
    --logging_dir /primus_oss/summary \
    --gradient_checkpointing true \
    --vit_gradient_checkpointing false \
    --warmup_ratio 0.03 \
    --deepspeed zero2 \
    --padding_free true \
    --attn_impl flash_attention_2 \
    --add_version False
```

---

### **Results**

| Model | VideoMME Performance | Notes |
|-------|---------------------|-------|
| **Original Checkpoint** | ~Baseline (close to released version) | 512 frames evaluation |
| **After SFT** | **-10 points** | Same 512 frames evaluation |

---

### **Hypothesis & Experiments Tried**

I suspect the **instruct model's capability is already too strong**, leading to **negative transfer** when fine-tuning on existing open-source datasets:

1. **Standard SFT** (`llava_video_165k`) → ❌ -10 points degradation
2. **Higher quality data** (`VideoR1` training set) → ❌ Still degradation  
3. **Removed `<think>` labels** from training data → ❌ Still degradation

---

Is Qwen3VL-8B-Instruct **already near its performance ceiling**, making additional SFT harmful?


---

**Any insights on fine-tuning strategies for already-strong instruct models would be greatly appreciated!** 🙏
